### PR TITLE
✨ add Hangman

### PR DESCRIPTION
**Pull request by LouisaStrunk02**
Changes:

add a hangman game

### CODE REVIEW

File: Hangman.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
+from HangmanStages import stages
+from Words import word_list
+import random
+
+def printHangmanStages(stage):
```
Comment (stefanluth) [inline Hangman.py:5]: Python uses snake_case for function names :)

File: Hangman.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    word_completion = "".join(word_as_list)
+
+    return word_completion
+
+def play(word):
+    remaining_tries = 7
+    word_was_guessed = False
+    hidden_letter = "_"
+    word_completion = hidden_letter * len(word)
+    listOfLetters = [0] * remaining_tries
```
Comment (stefanluth) [inline Hangman.py:39]: snake_case :)

File: Hangman.py [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+                print("Du hast " + guess + " schon einmal eingegeben.")
+            else:
+                listOfLetters[7 - remaining_tries] = guess
+                print(f"Das gesuchte Wort enthält den Buchstaben " + guess + " nicht.")
+                remaining_tries -= 1
+        else:
+            if guess == word:
+                print(guess)
+                word_was_guessed = True
+                break
```
Comment (stefanluth) [inline Hangman.py:58]: Does the word list include single letter "words"?

File: HangmanStages.py [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+                """
+                   --------
+                   |      |
+                   |      
+                   |    
+                   |      
+                   |     
+                   -
+                """
+    ]
```
Comment (stefanluth) [inline HangmanStages.py:65]: EOF

**Review by stefanluth — CHANGES_REQUESTED** [review]

**Review by DevBastiS — APPROVED** [review]

File: HangmanStages.py [hunk 5]
```diff
@@ -0,0 +1,66 @@
+stages = [
+                """
+                   --------
+                   |      |
+                   |      O
+                   |     \\|/
+                   |      |
+                   |     / \\
+                   -
+                """,
```
Comment (stefanluth) [inline HangmanStages.py:10]: Indentation is off :)

**Review by shoreal — APPROVED** [review]

**Review by stefanluth — APPROVED** [review]

**Pull request merged**